packages/lib-classifier/src/components/Classifier/components/MetaTools/components/FavouritesButton/FavouritesButton.js

```diff
@@ -0,0 +1,54 @@
+import counterpart from 'counterpart'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import theme from 'styled-theming'
+import { Icon } from 'grommet-icons'
+import zooTheme  from '@zooniverse/grommet-theme'
+import { PlainButton } from '@zooniverse/react-components'
+import HeartIcon from './HeartIcon'
+
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+export const Favourite = styled(HeartIcon)`
+  fill: none;
+  stroke: ${theme('mode', {
+    dark: zooTheme.dark.colors.font,
+    light: zooTheme.light.colors.font
+  })};
+  width: 1em;
+`
+
+export const IsFavourite = styled(Favourite)`
+  fill: #E45950;
+`
+
+export default function FavouritesButton (props) {
+  const { checked, mode, onClick } = props
```

Comment:
Leftover `mode` here. I don't fully understand how the theme is passed down, but the mode prop doesn't seem to be actually used by the component.

Reply:
Yeah, I'm going to need to look into where ThemeProvider should be set, it turns out there isn't one in `_app.js` but probably should be

Reply:
The `theme()` function here is spooky magic to me. I just copied it from another component.